build/common.sh

```diff
@@ -272,7 +272,7 @@ function kube::build::docker_image_exists() {
 
   # We cannot just specify the IMAGE here as `docker images` doesn't behave as
   # expected.  See: https://github.com/docker/docker/issues/8048
-  "${DOCKER[@]}" images | grep -Eq "^${1}\s+${2}\s+"
+  "${DOCKER[@]}" images | grep -Eq "${1}\s+${2}\s+"
```

Comment:
Nevermind - you said it in your comment.  I am not seeing docker.io - what docker version is doing this?